DigitalLearningSolutions.Web/Views/TrackingSystem/DelegateApprovals/Index.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@model DelegateApprovalsViewModel
+
+@{
+  ViewData["Title"] = "Approve delegate registrations";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Approve delegate registrations</h1>
+    @if (Model.Delegates.Any()) {
+      @foreach(var delegateApproval in Model.Delegates)
+      {
+        <partial name="_delegateExpandable" model="delegateApproval" />

```

Comment:
Could this partial be called _approveDelegateExpandable (we will have other delegate expandables unrelated to the approval functionality)